api/data_explorer/controllers/export_url_controller.py

```diff
@@ -259,15 +279,34 @@ def _get_filter_query(filters):
     return query
 
 
+def _get_doc_generator(filter_arr):
```

Comment:
I think we should query BQ instead of ES. The code will be easier to understand, and also the columns would be in the right order. (Right now columns are in random order.) Note this code will go away once Saturn UI can run SQL queries.

I have some ideas for addressing slowness:
- If user selected cohort, kick off creating GCS file when they click export button, instead of when they save cohort name dialog
- Could build JSON file representing everything on page load

Reply:
Per offline discussion, this would likely be quite slow. However this has all been moved to the indexer, so doesn't really matter.

Reply:
Sorry that's not totally true, we still query ES to get the sample_set.